scanners/boostsecurityio/supply-chain-inventory/module.yaml

```diff
@@ -0,0 +1,23 @@
+api_version: 1.0
+
+id: boostsecurityio/supply-chain-inventory
+name: BoostSecurity Supply Chain Inventory
+namespace: boostsecurityio/supply-chain-inventory
+scan_types:
+  - metadata
+  - sci
+
+config:
+  support_diff_scan: false
+  require_full_repo: true
+
+steps:
+  - scan:
+      format: supply_chain_inventory
+      command:
+        docker:
+          image: public.ecr.aws/boostsecurityio/boost-scanner-composition:f9c5372@sha256:30b885b536fa0382b51c15ba4f1d8307bf997d7d6ef1a151ad4b4ca9f7b00c63
```

Comment:
@ledo01 Looks like you want to merge a dev-registry branch as opposed to `main`
That's not the "normal" process. 
I see that this ^^ is in `main` , plus some output script Dylan added, that's fine...

Also I cannot correlate Docker `f9c5372` with `30b885b536fa0382b51c15ba4f1d8307bf997d7d6ef1a151ad4b4ca9f7b00c63` here 

https://app.circleci.com/pipelines/github/boostsecurityio/boostsec-scanner-composition/40/workflows/0f2485a3-5479-4576-977a-1faabba78afc/jobs/82

Where did you get the sha256 for the Docker ?

Reply:
> Looks like you want to merge a dev-registry branch as opposed to main
That's not the "normal" process.
I see that this ^^ is in main , plus some output script Dylan added, that's fine...

Yes, I didn't want to merge @Dyc0de changes, only mines, this is why I created a branch without his commit based on main. If his changes are ready, I can reopen the PR with it. 

> Also I cannot correlate Docker f9c5372 with 30b885b536fa0382b51c15ba4f1d8307bf997d7d6ef1a151ad4b4ca9f7b00c63 here

I got it from this step: https://app.circleci.com/pipelines/github/boostsecurityio/boostsec-scanner-composition/40/workflows/0f2485a3-5479-4576-977a-1faabba78afc/jobs/82/parallel-runs/0/steps/0-118

Also, the scanner has been used in dev for more than a week now, so I am assuming this is the right sha.

Reply:
yupyup, makes sense